en_ewt-ud-train.conllu

```diff
@@ -31082,7 +31082,7 @@
 36	Governments	government	NOUN	NNS	Number=Plur	33	nmod	33:nmod:of	_
 37	to	to	ADP	IN	_	39	case	39:case	_
 38	various	various	ADJ	JJ	Degree=Pos	39	amod	39:amod	_
-39	NGO's	ngo'	NOUN	NNS	Number=Plur	33	nmod	33:nmod:to	_
+39	NGO's	ngo	NOUN	NNS	Number=Plur	33	nmod	33:nmod:to	_
```

Comment:
NGO

Reply:
This one has no other examples in the treebank, but it is also not a proper noun, which would make me think it could follow the other conventions